<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="The-Batcomputer" failures="1" tests="2" name="Tests" time="10.934" errors="0" timestamp="2021-12-07T14:49:44 EST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.832">
    <failure type="java.lang.AssertionError" message="Output: 4 is not correct expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Output: 4 is not correct expected [true] but found [false]
at stepDefinitions.TestStepDefs.verify_the_output_is(TestStepDefs.java:59)
at âœ½.verify the output is 4(file:///C:/Users/dylan/Desktop/Everything/Personal/Coding/Eclipse%20Workspaces/Automation%20Framework/SeleniumTests/src/test/java/features/TestFeature.feature:7)
... Removed 3 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.667"/>
</testsuite> <!-- Tests -->
